File: Python-Assignment/Chapter-Three/Question8.java
import java.util.Scanner;

public class Question8{
   public static void main(String []args){

	Scanner input = new Scanner(System.in);

	double minNumber = Double.POSITIVE_INFINITY;
	int maxNumber = 0;
	int total = 0;
	int count = 0;

	for (int index = 0; index < 4; index++){
		System.out.print("Enter a number: ");
		int number = input.nextInt();

		if (number < minNumber){
			minNumber = number;
		} 
		if (number > maxNumber){
			maxNumber = number;
		}
		total = total + number;
		count++;

	}

	int average = total / count;

	System.out.println("The sum of all number is: " + total);
	System.out.println("The average is: " + average);
	System.out.println("The minimum number is: " + minNumber);
	System.out.println("The maximum number is: " + maxNumber);
   }
}

File: Python-Assignment/chapterThree/question8.py
max_number = 0
min_number = float('inf')
count = 0
total = 0

for index in range(0, 4):
	number = int(input("Enter a number: "))
	if number < min_number:
		min_number = number
	if number > max_number:
		max_number = number
	total = total + number
	count = count + 1

average = total / count
print("The sum of all numbers: ", total)
print("The average: ", average)
print("The lowest number is: ", min_number)
print("The highest number is: ", max_number)
